private/buf/bufsync/syncer.go

```diff
@@ -512,49 +527,30 @@ func (s *syncer) isGitCommitSynced(ctx context.Context, moduleIdentity bufmodule
 	return commitSynced, nil
 }
 
-// readModuleAt returns a module that has a name and builds correctly given a commit and a module
-// directory, or a read module error. If the module builds, it might be returned alongside a not-nil
-// error.
+// readModule returns a module that has a name and builds correctly given a commit and a module
+// directory. If the module builds, it might be returned alongside a non-nil error.
```

Comment:
```suggestion
// directory.
```

That's no longer true.